ta/pkcs11/src/pkcs11_token.c

```diff
@@ -25,13 +27,12 @@
 /* Static allocation of tokens runtime instances (reset to 0 at load) */
 struct ck_token ck_token[TOKEN_COUNT];
 
-/* Static allocation of tokens runtime instances */
 struct ck_token *get_token(unsigned int token_id)
 {
-	if (token_id > TOKEN_COUNT)
-		return NULL;
+	if (token_id < TOKEN_COUNT)
+		return &ck_token[confine_array_index(token_id, TOKEN_COUNT)];
```

Comment:
Maybe prefer here `return ck_token + confine_array_index(token_id, TOKEN_COUNT);`? :)

Reply:
Sure, but some seem to prefer this format so I'm fine with either way.

Reply:
ok, leave as is.